src/auto-evo/RunResults.cs

```diff
@@ -107,13 +107,22 @@ public void ApplyResults(GameWorld world, bool skipMutations)
         ///     Throws an exception if no population is found
         ///   </para>
         /// </remarks>
-        public long GetGlobalPopulation(Species species)
+        public long GetGlobalPopulation(Species species, bool resolveMoves = false)
         {
             long result = 0;
 
             foreach (var entry in results[species].NewPopulationInPatches)
             {
-                result += Math.Max(entry.Value, 0);
+                var population = entry.Value;
+
+                if (resolveMoves){
+                    foreach (var migration in results[species].SpreadToPatches)
+                    {
+                        population += migration.Population;
```

Comment:
This is not right. Only the migrations that affect the current patch in entry should be considered.
Also the migration has two parts, the to and from patch which determine if the population should be subtracted or added.